lib/logstash/outputs/datadog_metrics.rb

```diff
@@ -73,7 +72,7 @@ def receive(event)
 
     dd_metrics = Hash.new
     dd_metrics['metric'] = event.sprintf(@metric_name)
-    dd_metrics['points'] = [[to_epoch(event.timestamp), event.sprintf(@metric_value).to_f]]
+    dd_metrics['points'] = [[event.timestamp.to_i, event.sprintf(@metric_value).to_f]]
```

Comment:
There is nothing wrong with using the ```to_epoch``` method to coerce the "timestamp" given.  It it simply implemented wrong.